src/foam/nanos/auth/AgentUserAuthService.java

```diff
@@ -43,7 +43,7 @@ public void start() {
   */
   public User actAs(X x, User entity) throws AuthenticationException {
     User agent = (User) x.get("user");
-    User user = (User) userDAO_.find(entity.getId());
+    User user = (User) userDAO_.find_(x, entity.getId());
 
```

Comment:
I feel like entity might be a clearer name than user for this variable?

Reply:
Sure, but entity will become the user within the context. I envisioned it as a way to preempt that idea.